app/src/visionglass/AndroidManifest.xml

```diff
@@ -8,11 +8,17 @@
     <application>
         <activity
             android:name=".VRBrowserActivity"
-            android:screenOrientation="portrait">
+            android:screenOrientation="portrait"
+            android:exported="true">
```

Comment:
Hmm, isn't this only for allowing external apps to call the activity? Shouldn't be needed for an activity from the same app id. Or maybe I am wrong

Reply:
This fixes a warning, apparently the main activity should have `exported=true`.